src/components/quickStarts.stories.mdx

```diff
@@ -0,0 +1,470 @@
+<Meta title="Quick Starts" />
+
+# Getting Started
+
```

Comment:
Can we add links to other parts of this page so that I can skip to it from here?

Reply:
If not, its okay I guess

Reply:
I tried the markdown syntax, it doesn't work in storybook. 
According to storybook website, we may need to add some 'addons' to achieve that. It takes some time cause their doc seems not very good, it's hard to find an example.
Since I'm working on downlink policy issue, I don't have extra time to do this right now. We could improve this later if you agree.

Reply:
Will add table of content as well as how-to for data message in the future